runelite-api/src/main/java/net/runelite/api/MenuAction.java

```diff
@@ -139,7 +139,40 @@
 	 * Menu action injected by runelite for its menu items.
 	 */
 	RUNELITE(1500),
-	
+
+	/**
+	 * Menu action triggered by taking an object
+	 */
+	TAKE_OBJECT_GROUND(20),
+
+	/**
+	 * Menu actions triggered by calling for collector in Barbarian Assault.
+	 */
+	TELL_RED(33),
```

Comment:
33, 34, 35, and 36 should just be FIRST/SECOND/THIRD/FOURTH_ITEM_ACTION. They are different per-object and they should not all be here. Although there are some others here already.